playbooks/templates/rax-maas/lb_api_check_heat_cfn.yaml.j2

```diff
@@ -6,11 +6,11 @@ period            : "{{ maas_check_period_override[label] | default(maas_check_p
 timeout           : "{{ maas_check_timeout_override[label] | default(maas_check_timeout) }}"
 disabled          : "{{ (check_name | match(maas_excluded_checks_regex)) | ternary('true', 'false') }}"
 target_resolver   : "IPv4"
-target_hostname   : "{{ maas_external_ip_address }}"
+target_hostname   : "{{ internal_vip_address }}"
 details           :
-    url           : "{{ maas_heat_cfn_scheme | default(maas_scheme)}}://{{ maas_external_ip_address }}:8000"
+    url           : "http://{{ internal_vip_address }}:8000"
 monitoring_zones_poll:
-{% for zone in maas_monitoring_zones %}
```

Comment:
Why are we switching everything to the private zones, are we killing the old ones?